### PR TITLE
Add new Jumper AION 2400 target

### PR DESCRIPTION
**Pull request by norulers**

### CODE REVIEW

File: src/hardware/RX/Jumper AION Nano RX.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "power_txen": 10,
+    "power_min": 0,
+    "power_high": 3,
+    "power_max": 3,
+    "power_default": 3,
+    "power_control": 0,
+    "power_values": [-10,-6,-3,1],
+    "button": 0,
+    "led": 16
+}
```
Comment (pkendall64) [inline src/hardware/RX/Jumper AION Nano RX.json:22]: This file is not required, it's is exactly the same as the BETAFPV nano file.

File: src/hardware/targets.json [hunk 2]
```diff
@@ -351,12 +351,26 @@
...
                 "lua_name": "AION 2G4NANO TX",
                 "layout_file": "Jumper AION Nano.json",
                 "upload_methods": ["uart", "wifi"]
             }
+        },
+        "rx_2400": {
+            "nano": {
+                "product_name": "Jumper AION Nano 2.4GHz RX",
+                "lua_name": "AION 2G4NANORX",
+                "layout_file": "Jumper AION Nano RX.json",
```
Comment (pkendall64) [inline src/hardware/targets.json:371]: This is reference the BETAFPV Nano json file.
I think we will be renaming the files to be more Generic later in another PR.

File: src/targets/Jumper_2400.ini [hunk 3]
```diff
@@ -14,6 +14,17 @@ extends = env:Jumper_AION_2400_T-Pro_TX_via_UART
...
+extends = env:Unified_ESP32_2400_TX_via_UART
+board = pico32
+board_config = jumper.tx_2400.t-lite
+
+[env:Jumper_AION_2400_T-Lite_TX_via_WIFI]
+extends = env:Jumper_AION_2400_T-Lite_TX_via_UART
+
+[env:Jumper_AION_2400_T-Lite_TX_via_ETX]
+extends = env:Jumper_AION_2400_T-Lite_TX_via_UART
+
```
Comment (pkendall64) [inline src/targets/Jumper_2400.ini:27]: These are no longer required we you can select the Unified TX target and then choose the layout to attach to the firmware at the end of the build process.

File: src/targets/Jumper_2400.ini [hunk 4]
```diff
@@ -24,3 +35,16 @@ extends = env:Jumper_AION_NANO_2400_TX_via_UART
...
+board_config = jumper.rx_2400.nano
+
+[env:Jumper_AION_Nano_2400_RX_via_BetaflightPassthrough]
+extends = env:Jumper_AION_Nano_2400_RX_via_UART
+upload_protocol = custom
+upload_speed = 420000
+upload_command = ${env_common_esp82xx.bf_upload_command}
+
+[env:Jumper_AION_Nano_2400_RX_via_WIFI]
+extends = env:Jumper_AION_Nano_2400_RX_via_UART
```
Comment (pkendall64) [inline src/targets/Jumper_2400.ini:50]: These are no longer required we you can select the Unified TX target and then choose the layout to attach to the firmware at the end of the build process.

File: src/targets/Jumper_2400.ini (hunk 3 above)
Comment (jurgelenas) [reply src/targets/Jumper_2400.ini:27]: Do we expect users to manually add required config files?

Comment (pkendall64) [reply src/targets/Jumper_2400.ini:27]: No, not at all. Hopefully, when we release Configurator will be fully supporting selecting targets from the targets.json file. Then these PIO based targets become pretty useless.
If configurator will not be ready then we can add these targets to PIO. The problem is that there are so many now and the number is growing at an alarming rate!

File: src/hardware/RX/Jumper AION Nano RX.json (hunk 1 above)
Comment (pkendall64) [reply src/hardware/RX/Jumper AION Nano RX.json:22]: We've done some refactoring and now the base target file should be `Generic 2400 PA.json`

**Comment by pfeerick** [comment]
Should this work atm with the TLite V2 and ETX passthrough (running nightly that supports the V2) when flashed via PIO? Just wondering if I should be looking at a problem on the ETX side or ELRS side... esptool is failing to connect, but passthrough *appears* to be starting.

**Comment by pkendall64** [comment]
> Should this work atm with the TLite V2 and ETX passthrough (running nightly that supports the V2) when flashed via PIO? Just wondering if I should be looking at a problem on the ETX side, ELRS side or something with the hardware they sent me ;) ... esptool is failing to connect, but passthrough _appears_ to be starting.

Yes, this does work. I have my Jumper TLite V2 with EdgeTX PR2046 flashed and this flashed find using via ETX flashing method.

**Review by pkendall64 — APPROVED** [review]

**Review by JyeSmith — APPROVED** [review]

**Pull request merged**